File: Fibanacci sequence.js
var a=[]
var input =prompt("enter the length of array:")


function Positive(input)
{
    if (input>0)
        return true
    
    else
        return false
}

function List( input)
{
    if (Positive(input))
    {
        if (input==1)
        {
            a=[0]
        }
        else if (input==2)
        {
            a=[0,1]
        }
        else if (input>2)
        {
            a=[0,1]
            while (a.length< input){
            var num1=a[a.length-1]
            var num2=a[a.length-2]
            a.push (num1+num2)
            }
            return a
        }
    }
    else 
    {
        alert("Please enter positive number")
    }
}
alert(List( input))

File: Fibanacci sequence.py
import re


a=[]
num2=input("Enter the length of the array: ")
num=int(num2)
def Positive(num):
    if num>0:
        return True
    else:
        return False

def list_num(num):
    if Positive(num) :
        if num==1 :
            a=[0]
        elif num==2:
            a=[0,1]
        elif num>2:
            a=[0,1]
            while len(a) < num :
                num1=a[len(a)-1]
                num2=a[len(a)-2]
                a.append(num1+num2)
            return a
    else :
        print ("Please enter a number more than 0 :)")

print (list_num(num))